doc/adr/2019-02-13-preserve-all-functionality.md

```diff
@@ -0,0 +1,31 @@
+**Title**: Preserve all functionality offered by the raw API.
+
+**Status**: proposed
+
+**Context**: When we design APIs for the client libraries one may need to
+tradeoff ease of use for functionality. All things being equal it is easier to
+design simpler, easier to understand client libraries if one is willing to
+remove functionality that is rarely used. While we want to encourage users to
+adopt the libraries, we do not want to abandon advanced users to their own
+devices. We do not want them to be tempted or forced to discard the client
+libraries in favor of raw REST or gRPC because that is the only way they can
+access a feature.
+
+**Decision**: the libraries must preserve all functionality. If it is possible
```

Comment:
Nit: Be consistent with whether you capitalize the first word after the colon.

Reply:
Done.